docs/runtimes/job-function.md

```diff
@@ -49,6 +49,21 @@ To run the job:
 project.run_function("train")
 ```
 
+You can configure automatic retries for failed job runs by passing a `retry` dictionary
+when running a function. This enables MLRun to handle transient errors such as runtime failures,
```

Comment:
when running a function. This enables MLRun to automatically handle transient errors such as runtime failures,
OOM, or pod eviction.